icubam/backoffice/handlers/upload.py

```diff
@@ -29,7 +29,8 @@ def post(self):
     sync = synchronizer.CSVSynchcronizer(self.db)
```

Comment:
it seems like this is not the same name as in synchronizer anymore.

Reply:
Does it mean this endpoint is not tested?

Reply:
shhhh